codemod/base.py

```diff
@@ -322,6 +326,11 @@ def __init__(self,
         @param inc_extensionless    If True, will include all files without an
                                     extension when checking
                                     against the path_filter
+        @param match_pattern        String representation of the match pattern,
+                                    for use by spotlight_util if --spotlight is
+                                    set.
+        @param spotlight            If True, will use spotlight search to save
+                                    time. Requires match_pattern to be set.
```

Comment:
It seems like you should be able to combine these two parameters, no? Perhaps rename to `spotlight_match_pattern`?